pkg/controller/attach_detach.go

```diff
@@ -242,7 +244,7 @@ func (r *ReconcileAttachDetach) triggerAttach(ctx context.Context, azVolumeAttac
 			}
 		}
 
-		_ = azureutils.UpdateCRIWithRetry(cloudCtx, nil, r.controllerSharedState.cachedClient, r.controllerSharedState.azClient, azVolumeAttachment, updateFunc, consts.ForcedUpdateMaxNetRetry)
+		_ = azureutils.UpdateCRIWithRetry(updateCtx, nil, r.controllerSharedState.cachedClient, r.controllerSharedState.azClient, azVolumeAttachment, updateFunc, consts.ForcedUpdateMaxNetRetry)
```

Comment:
Move the declaration and initialization of `updateCtx` to just before this line or simply use `context.Background()` directly. Include an explanatory comment about the reason to use a separate context here.

Reply:
👍